iron-overlay-behavior.html

```diff
@@ -391,6 +428,18 @@
       }
     },
 
+    _onCaptureFocus: function (event) {
+      if (this._manager.currentOverlay() === this &&
+          this.withBackdrop) {
+        if (Polymer.dom(event).path.indexOf(this) === -1) {
+          event.stopPropagation();
```

Comment:
there is a specific line in the tests for this https://github.com/PolymerElements/iron-overlay-behavior/pull/92/files#diff-19c5ed5153f2ca6090daf300984fcad2R505